tests/unit/test_speech.py

```diff
@@ -347,3 +352,13 @@ def test_languageDetection(self):
 			beepForCapitals=False,
 		)
 		self.assertEqual(repr(list(output)), expected)
+
+
+class SPeechExtensionPoints(unittest.TestLoader):

```

Comment:
Lower case 'p' in the class name, and correct base class:

```suggestion
class SpeechExtensionPoints(unittest.TestCase):
```

Reply:
Thanks @lukaszgo1!